sdk/deepbook/.gitignore

```diff
@@ -8,6 +8,8 @@ lerna-debug.log*
 .pnpm-debug.log*
 .DS_Store
 .idea/
+src/*.js
```

Comment:
This shouldn't be needed unless you're building into the source directory which we shouldn't be doing

Reply:
fixed